<?xml version="1.0" encoding="UTF-8"?><record_update table="sys_rest_message_fn">
    <sys_rest_message_fn action="INSERT_OR_UPDATE">
        <authentication_type>inherit_from_parent</authentication_type>
        <basic_auth_password/>
        <basic_auth_profile/>
        <basic_auth_user/>
        <content/>
        <function_name>Get Restore Point by ID</function_name>
        <http_method>get</http_method>
        <lock>false</lock>
        <oauth2_profile/>
        <protocol_name/>
        <qualified_name>Veeam Server [ Get Restore Point by ID ]</qualified_name>
        <rest_endpoint>${baseUrl}/api/v1/objectRestorePoints/${id}</rest_endpoint>
        <rest_message display_value="Veeam Server">270ffe5b97d91d10628271e3a253afe8</rest_message>
        <sys_class_name>sys_rest_message_fn</sys_class_name>
        <sys_created_by>c.girbu</sys_created_by>
        <sys_created_on>2022-10-28 11:43:35</sys_created_on>
        <sys_id>4ff1eff11bfe1910e06d520b234bcb04</sys_id>
        <sys_mod_count>2</sys_mod_count>
        <sys_name>Get Restore Point by ID</sys_name>
        <sys_package display_value="Veeam App" source="x_tesgh_veeam_app">d50434871b9dd91063fd6243604bcb7b</sys_package>
        <sys_policy/>
        <sys_scope display_value="Veeam App">d50434871b9dd91063fd6243604bcb7b</sys_scope>
        <sys_update_name>sys_rest_message_fn_4ff1eff11bfe1910e06d520b234bcb04</sys_update_name>
        <sys_updated_by>pavel.egorov</sys_updated_by>
        <sys_updated_on>2023-08-11 11:07:04</sys_updated_on>
        <use_basic_auth>false</use_basic_auth>
        <use_mid_server/>
        <use_mutual_auth>false</use_mutual_auth>
    </sys_rest_message_fn>
</record_update>
